### ISSUE TITLE
Botfront up: Couldn't find env file

### ISSUE THREAD

**Issue by GeovanaRamos** [botfront/botfront#446]
**Describe the bug**
After command "botfront up" an error occurs with below messages:
`✖ Couldn't start Botfront. Retrying in verbose mode...
⠧ Starting Botfront...The IMAGES_CURRENT_RASA variable is not set. Defaulting to a blank string.
The IMAGES_CURRENT_ACTIONS variable is not set. Defaulting to a blank string.
Couldn't find env file: /home/geovana/code/t2/.env
`

**To Reproduce**
1. "botfront init" on a new directory
2. Create in current directory
3. botfront up

**Additional context**
Botfront latest version

**Comment by znat** [comment]
Did you recently upgrade Botfront? Have you looked at the migration guide? https://botfront.io/docs/migration/

**Comment by GeovanaRamos** [comment]
Yes, but in this case, I was starting a project from scratch to test the new version, using cli. If I build using my own docker-compose botfront works.

**Comment by znat** [comment]
Can you attach a screenshot of your folder content and the output of `botfront -v`?

**Comment by GeovanaRamos** [comment]
Sure! I followed the steps again.

![image](https://user-images.githubusercontent.com/31005087/73074447-6cd3ea80-3e98-11ea-9d19-4093ab4b6f3d.png)

**Comment by GeovanaRamos** [comment]
Forgot folder content
![image](https://user-images.githubusercontent.com/31005087/73075029-c0930380-3e99-11ea-950c-d56c4f0ed4ae.png)

**Comment by znat** [comment]
Can you confirm your project has the following structure: https://botfront.io/docs/rasa/custom-actions/#development

When you run `botfront up` it generates a `docker-compose.yml` file and a `.env` files at the root of your project (which seems not to be the case based on your logs).
The question is why isn't it there.
I think we can exclude the permission issue since the project folder is copied without any problem.

Is it possible that you have conflicting versions? are you using nvm? Did you install the cli from the source with `npm link`?


I suspect you have conflicting versions installed on your computer

**Comment by GeovanaRamos** [comment]
It seems you were right. Tried in another computer with a clean install and the project folder was created as it should be. Just to register for anyone who has the same problem, I updated to the new version using npm install -g botfront and had the problem. Tried with sudo and updated just fine. Problem solved! Thanks for the support.  We're loving Botfront.

**Comment by MatthieuJnon** [comment]
Hi, just so you know, you shouldn't have to use sudo with npm. If you have to, it's symptomatic of a larger problem with you npm installation. Refer to this documentation on how to fix it. https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally

**Comment by pranavinu** [comment]
Hi, I am also facing the same issue after upgrading the bot front in windows. Can anyone suggest me how to solve the issue. Please see the screenshot for more information. Thank You
@znat Please look into this. Thank you.

<img width="675" alt="Botfront" src="https://user-images.githubusercontent.com/22297243/73171193-28f61500-4126-11ea-873a-696eff11c2cc.PNG">

**Comment by znat** [comment]
It seems that upgrading the cli with npm install is not enough and let
conflicting versions in place. Try uninstalling and reinstalling the cli

Le lun. 27 janv. 2020 06 h 28, pranavi padegal <notifications@github.com> a
écrit :

**Comment by pranavinu** [comment]
Yup, that worked for me Znat. But came up with another  issue. Please see the following link. https://github.com/botfront/botfront/issues/399#issuecomment-579175623

**Comment by pythiantech** [comment]
I am on macOS Catalina 10.15.6 and facing the same issue. I have tried uninstalling and reinstalling the cli with no joy. Also uninstalled node completely and tried again. But the same issue persists.
